package.json

```diff
@@ -94,17 +94,21 @@
         "@typescript-eslint/eslint-plugin": "^6.0.0",
         "@typescript-eslint/parser": "^6.0.0",
         "@vscode/test-electron": "^2.3.0",
-        "@vscode/vsce": "2.24.0",
+        "@vscode/vsce": "2.25.0",
         "eslint": "8.43.0",
         "glob": "^8.1.0",
         "lerna": "^7.0.0",
         "mocha": "^10.2.0",
+        "ovsx": "0.9.2",
```

Comment:
should it be `"osvx": "^0.9.2"`? or just `"osvx": "^0.9.0"` instead of fixed version?

Reply:
Normally yes, that's the pattern we tend to use, that permits to easily (yarn) upgrade to new versions of dependencies that should remain compatible (no major version increment). We could do it here too and it would likely work ok. 

The reason I used a fixed version is to prevent a case that may not be very likely: if `ovsx` were to publish a new version, in which they make changes that are only compatible with `"@vscode/vsce": ">2.25.0"`, that would clash with our local pinning (resolutions block entry) of that dependency to v2.25.0. 

By using `"ovsx": "0.9.2"`, we insure that we use that specific version until further notice, and avoid the potential issue above, creeping-in on us through a "yarn upgrade".

Makes sense?

Reply:
Makes sense. Thanks for the explanation.